### PR TITLE
Bugfix/cmake config binaries

### PR DESCRIPTION
**Pull request by inbangsa**
This local for github actions

### CODE REVIEW

File: cmake/modules/ifm3d-config.cmake.in [hunk 1]
```diff
@@ -26,7 +26,12 @@ else()
             find_package(OpenCV 2.4 REQUIRED COMPONENTS core)
           else()
             find_package(OpenCV REQUIRED COMPONENTS core)
-          endif()
+          endif()  
+		  find_package(PCL 1.7 REQUIRED COMPONENTS common)
```
Comment (graugans) [inline cmake/modules/ifm3d-config.cmake.in:30]: Above we do install 1.10 isn't this an issue?

File: modules/image/src/libifm3d_image/CMakeLists.txt [hunk 2]
```diff
@@ -56,7 +63,8 @@ target_link_libraries(
   PUBLIC
     ifm3d_framegrabber
     ifm3d_camera
-    ${OpenCV_LIBRARIES}
+    ${OpenCV_LIBS}
```
Comment (graugans) [inline modules/image/src/libifm3d_image/CMakeLists.txt:66]: May I ask what is the difference between ``OpenCV_LIBRARIES`` and ``OpenCV_LIBS``?

File: cmake/modules/ifm3d-config.cmake.in (hunk 1 above)
Comment (inbangsa) [reply cmake/modules/ifm3d-config.cmake.in:30]: ifm3d dependens on PCL for compile time and not on link time, so this is just to satisfy the compiler. when user will link with its PCL The required header file will be used from user PCL.

File: modules/image/src/libifm3d_image/CMakeLists.txt (hunk 2 above)
Comment (inbangsa) [reply modules/image/src/libifm3d_image/CMakeLists.txt:66]: Standard find_package call export package_INCLUDE_DIRS and package_LIBRARIES but opencv config file documenation says its export OpenCV_LIBS. if we print the values they both show same values. All the opencv example uses  OpendCV_LIBS

**Comment by graugans** [comment]
Can you please stash the 26 commits :-)

**Pull request merged**